pom.xml

```diff
@@ -536,8 +536,8 @@
                             <target>
                                 <mkdir dir="${project.build.directory}/extra-resources"/>
                                 <mkdir dir="${project.build.directory}/tmp"/>
-                                <exec executable="bash" output="${project.build.directory}/extra-resources/rapids4spark-version-info.properties">
-                                    <arg value="${project.basedir}/../build/build-info"/>
+                                <exec executable="bash" failonerror="true" output="${project.build.directory}/extra-resources/rapids4spark-version-info.properties">
+                                    <arg value="${user.dir}/build/build-info"/>
```

Comment:
`user.dir` seems inapproprite here.  Shouldn't it be `${project.basedir}/build/build-info` instead?  Maven guarantees `project.basedir` evaluates to the directory where pom.xml resides which is what we want.  `user.dir` is less controllable.